### PR TITLE
refactor/naming starboard to trivy operator

### LINKED ISSUE #1162: rename starboard to trivy-operator
it is required to perform naming refactor from `Starboard` to `Trivy-Operator`, changes related to :
 - starboard operator code
 - starboard operator docs
 - starboard cli is out of scope (#1168 )

### PR DESCRIPTION
**Pull request by chen-keinan**
## Description
 Naming refactor from Starboard to Trivy-Operator
## Related issues
- Close #1162 

Remove this section if you don't have related PRs.

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/starboard/blob/main/CONTRIBUTING.md) to this repository.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/starboard/tree/main/docs) with the relevant

### CODE REVIEW

**Review by josedonizetti — COMMENTED** [review]
@chen-keinan Left two comments, and also think we need to rename the package imports on:

- https://github.com/aquasecurity/starboard/blob/main/pkg/report/templates/workload_report.qtpl
- https://github.com/aquasecurity/starboard/blob/main/pkg/report/templates/node_report.qtpl

- https://github.com/aquasecurity/starboard/blob/main/pkg/report/templates/namespace_report.qtpl

Besides it, the tests I did locally were all working.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,11 +70,11 @@ Each commit message doesn't have to follow conventions as long as it is clear an
...
-| `starboard`              | `docker.io/aquasec/starboard:dev`              | Starboard command-line interface                              |
-| `starboard-operator`     | `docker.io/aquasec/starboard-operator:dev`     | Starboard Operator                                            |
-| `starboard-scanner-aqua` | `docker.io/aquasec/starboard-scanner-aqua:dev` | Starboard plugin to integrate with Aqua vulnerability scanner |
+| Binary               | Image                                          | Description                                                   |
+|----------------------|------------------------------------------------|---------------------------------------------------------------|
+| `starboard`          | `docker.io/aquasec/starboard:dev`              | Starboard command-line interface                              |
+| `trivy-operator`     | `docker.io/aquasec/trivy-operator:dev`         | Trivy Operator                                                |
+| `trivy-scanner-aqua` | `docker.io/aquasec/starboard-scanner-aqua:dev` | Starboard plugin to integrate with Aqua vulnerability scanner |
```
[the remark this answers is not in the document]
Comment (chen-keinan) [reply CONTRIBUTING.md:77]: this is not the operator; the `starboard-scanner-aqua`  will be removed with the next PR together with CLI

File: pkg/cmd/root.go [hunk 2]
```diff
@@ -38,11 +38,11 @@ $ %[1]s get vulnerabilityreports deployment/nginx -o yaml
 `
 )
 
-func NewRootCmd(buildInfo starboard.BuildInfo, args []string, outWriter io.Writer, errWriter io.Writer) *cobra.Command {
+func NewRootCmd(buildInfo trivyoperator.BuildInfo, args []string, outWriter io.Writer, errWriter io.Writer) *cobra.Command {
 	var cf *genericclioptions.ConfigFlags
 
 	rootCmd := &cobra.Command{
-		Use:           "starboard",
+		Use:           "trivyoperator",
```
[the remark this answers is not in the document]
Comment (chen-keinan) [reply pkg/cmd/root.go:45]: CLI is out of scope for this PR , as its going to be remove with next PR

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chen-keinan) [reply CONTRIBUTING.md:77]: regarding qtpl files , the report are part of CLI  , which is going to be removed with next PR

**Review by josedonizetti — APPROVED** [review]

**Comment by chen-keinan** [comment]
Closing the PR. A new repo trivy-operator has been added

**Pull request closed without merging**